### PR TITLE
Dump factor, offset, min, max as float. Load as numeric

### PR DESCRIPTION
**Pull request by chrisoro**
https://github.com/ebroecker/canmatrix/issues/379

### CODE REVIEW

File: src/canmatrix/formats/cmjson.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def dump(db, f, **options):
     export_canard = options.get('jsonCanard', False)
     motorola_bit_format = options.get('jsonMotorolaBitFormat', "lsb")
     export_all = options.get('jsonAll', False)
+    native_types = options.get('jsonNativeTypes', False)
```
Comment (altendky) [inline src/canmatrix/formats/cmjson.py]:
```python
number_converter = float if native_types else str
```
Then below we can just unconditionally
```python
"factor": number_converter(signal.factor)
```
and similar.

File: src/canmatrix/formats/cmjson.py [hunk 2]
```diff
@@ -193,11 +194,12 @@ def load(f, **_options):
...
                     is_little_endian=is_little_endian,
                     is_signed=is_signed,
-                    is_float=is_float,
-                    factor=signal["factor"],
-                    offset=signal["offset"]
+                    is_float=is_float
                 )
 
+                new_signal.factor = new_signal.float_factory(signal["factor"])
+                new_signal.offset = new_signal.float_factory(signal["offset"])
```
Comment (altendky) [inline src/canmatrix/formats/cmjson.py]: Was `Signal` not properly applying the `float_factory` itself?

Comment (chrisoro) [reply src/canmatrix/formats/cmjson.py]: Oh, I didn't catch that. Thanks

File: src/canmatrix/formats/cmjson.py (hunk 1 above)
Comment (chrisoro) [reply src/canmatrix/formats/cmjson.py]: Sure.

**Pull request merged**